### DISCUSSION TITLE
should we show stake reward in the follow tables?

### DISCUSSION THREAD

**Discussion by lxm218** [tearust/tea-browser-wallet#119] [Ideas]
should we include stake reward in this table? currently, it only shows weight %
![image](https://user-images.githubusercontent.com/12605218/126539642-f891372d-654f-4098-8438-657983f338a4.png)

![image](https://user-images.githubusercontent.com/12605218/126539793-bb949e79-9907-49ff-81fe-14a24e103e8c.png)

**Comment by lxm218** [comment]
also show when my staking started

**Comment by kevingzhang** [comment]
we have shown the reward in the MyAssets. 
It is hard to have all detailed stored in the blockchain. Unless this is killer feature, I would suggest not to add it

**Comment by lxm218** [comment]
My Assets only shows total rewards, I cannot tell which staking gives me the higher return as each CML is different. I think this feature is important because investors need to know their return on each investment therefore they can make a decision if this is good investment and if they need to continue to stake or unstake, etc. This is like I have 10 different stocks, the exchange only shows my total gain/loss, I have no idea which stock has higher return than others.

**Comment by lxm218** [comment]
This table shows top mining CML list. Currently, it has all mining CMLs. Once mining CML increase, will we list them all, or just top 10 , top 20 based on rewards etc? 

![image](https://user-images.githubusercontent.com/12605218/126569005-989c6282-ffb9-4497-bf97-fef29ac6b259.png)

**Comment by kevingzhang** [comment]
我猜测最终的解决方案是另外一个独立的应用, 专门进行此类分析的应用. 那个应用并不存在在 Wallet 中. 我们希望 Wallet 更加简单直接. 专门的投资分析软件提供此类, 甚至于更加专业的分析功能. 那些应用是专门的 TAPP, 很有可能是收费软件, 可以参与投资和运营的. 我们自己也可以做, 以 Demo 的目的

**Comment by kevingzhang** [comment]
I would give this TApp a name:" Boba Insight".
It is a paid TApp (Rich dApp)providing data analysis service based on the user's data in the blockchain.
